matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/events/model/EventExt.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2022 The Matrix.org Foundation C.I.C.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.matrix.android.sdk.api.session.events.model
```

Comment:
Were you aware there was an existing `EventExt` in `the org.matrix.android.sdk.internal.session.events` package?

Reply:
Yes but it's internal, and toValidate is API

Reply:
Okay I see. Thanks for explaining.